### ISSUE TITLE
The process cannot access the file ....

### ISSUE THREAD

**Issue by onurcanyilmaz** [dotnet/sdk#15889]
SDK :  .NET 5.0.103
Hello,
I have a simple problem.
I have a DDD project with ASP.NET CORE 5
These are;  Api, Application,  Domain and EfCore

I took write a command "dotnet build".  
But terminal gave this error

_C:\Program Files\dotnet\sdk\5.0.103\NuGet.targets(131,5): error : The process cannot access the file 'C:\Users\username\.nuget\packages\microsoft.entityframeworkcore.tools\5.0.3\tools\net461\any\ef.exe' because it is being used by another process. [C:\Users\username\Source\repos\api.csproj_ 

I tried "dotnet clean" and "dotnet restore" but not successful
Do you have any help?
Thanks.

**Comment by AnakinSklavenwalker** [comment]
same issue here. never happened before. 

problem for me seems to be `dotnet clean` creates a bunch of `dotnet.exe` which stay open as background process.
Also this only seems to happen on Windows. Running everything via WSL ubuntu shell seems to work fine.

**Comment by onurcanyilmaz** [comment]
> same issue here. never happened before.
> 
> problem for me seems to be any MSBuild related command (e.g. clean/build/etc) creates a bunch of `dotnet.exe` (".NET Host" in taskmgr) which stay open as background process.
> Also this only seems to happen on Windows using Command Line. Running everything via Visual Studio or WSL ubuntu shell seems to work fine.
> 
> I'm running .NET 5 5.0.3 trying to build a simple console application.
> Build command: `dotnet build .\mySolution.sln -f net5.0 -o ./target/dotnet-executable`

Thanks @AnakinSklavenwalker 
My project created in visual studio code.

I dont have .sln file and i dont want to create .sln file.
Because other contributor does not have this problem
Do you have any other solution.

**Comment by AnakinSklavenwalker** [comment]
no i don't have any other project. 
it also seems i'm unable to reproduce this constantly. Either on new projects or even on my current project.

**Comment by wli3** [comment]
while we are looking into the root cause have you tried to use `dotnet build-server shutdown`?

**Comment by onurcanyilmaz** [comment]
Yes, we tried. But we got the same error.
Sometimes we got this error below.

Same SDK .NET 5.0.103

_C:\Program Files\dotnet\sdk\5.0.103\NuGet.targets(131,5): error : The process cannot access the file 'C:\Users\username \.nuget\packages\microsoft.extensions.apidescription.server\3.0.0\tools\net461\GetDocument.Insider.exe' because it is being used by another process. [C:\Users\users\Source\repos\project\WebApi\WebApi\WebApi.csproj]_

**Comment by AnakinSklavenwalker** [comment]
yes i tried `dotnet build-server shutdown`. it seems to work. but i wished this was not necessary.
